src/app/malaria-101/malaria-101.component.scss

```diff
@@ -0,0 +1,228 @@
+@import "../styles/main";
+
+video{
```

Comment:
1. file spacing
2. not all variables follow the scss standard
3. are you using `.mosquitoes` anywhere?
4. believe you've defined the `img` styling else where?
5. how responsive are the `.text-*` & `.arrow-*` & `.drop-*` classes?